README.md

```diff
@@ -67,8 +67,10 @@ cd tests
 Before running the tests, set the below environment variables. Use the [Developer Portal Manage API Keys page](https://developer.factset.com/manage-api-keys) to get these values.
 
 ```sh
-export ANALYTICS_API_USERNAME_SERIAL = "username-serial"
-export ANALYTICS_API_PASSWORD = "apikey"
+export ANALYTICS_API_DEFAULT_USERNAME_SERIAL = "default username-serial"
```

Comment:
Are we doing this change to use the QA username serials?

Reply:
Yes, the default and vault username/serials are the respective QA ones